ext/soap/php_encoding.c

```diff
@@ -1259,6 +1259,7 @@ static void model_to_zval_any(zval *ret, xmlNodePtr node)
 					ZVAL_NULL(&val2);
 					master_to_zval(&val2, get_conversion(XSD_ANYXML), node->next);
 					if (Z_TYPE(val2) != IS_STRING ||  *Z_STRVAL(val) != '<') {
+						Z_TRY_DELREF(val2);
```

Comment:
Hrm, in that case I'm wondering why the `zval_ptr_dtor(&val2)` four lines below cannot cause the same issue.